Update individual-orgs-summary.md (#21634)
DataDog/documentation: content/en/account_management/guide/csv_headers/individual-orgs-summary.md

kind: guide
---

> CSV header changes take effect the week of February 19, 2024. Below is an example of the new CSV structure. Use this table to update CSV headers your automation depends on:

Reviewer: ```suggestion
CSV header changes take effect the week of February 19, 2024. The following example demonstrates the new CSV structure. 

Use this table to update CSV headers that your automation depends on:
```
